src/pages/docs/[[...slug]].tsx

```diff
@@ -59,6 +60,8 @@ const mdxComponents = {
   Video,
   Link,
   a: Link,
+  aside: Callout,
```

Comment:
When copying and pasting a callout from Notion, it's converted to the HTML `aside` element. Figured it's worth supported both 

`<aside>my callout</aside>`
and
`<Callout>my callout</Callout>`

Let me know if it's better sticking with one.